design-documents/graph-ql/coverage/gift-wrapping.md

```diff
@@ -176,17 +196,27 @@ type CartItemUpdateInput {
     gift_wrapping_id: ID @doc(description: "The unique identifier of the gift wrapping to be used for the cart item")
     gift_message: GiftMessageInput @doc(description: "Gift message details for the cart item")
 }
+type RemoveItemFromCartInput {
+    gift_wrapping_id: ID @doc(description: "The unique identifier of the gift wrapping to be used for removing from the cart item")
+}
 
 type Mutation {
     setGiftOptionsOnCart(cart_id: String!, gift_message: GiftMessageInput, gift_wrapping_id: ID, gift_receipt_included: Boolean, printed_card_included: Boolean): SetGiftOptionsOnCartOutput @doc(description: "Set gift options like gift wrapping or gift message for the entire cart")
+    removeGiftOptionsOnCart(cart_id: String!, gift_wrapping_id: ID, gift_receipt_included: Boolean, printed_card_included: Boolean): RemoveGiftOptionsOnCartOutput @doc(description: "Remove gift options gift wrapping for the entire cart")
```

Comment:
Based on recent agreement we must always use `input` wrapper for mutation arguments.